modules/monitoring-configuring-notifications-for-default-platform-alerts.adoc

```diff
@@ -42,37 +42,40 @@ $ oc -n openshift-monitoring get secret alertmanager-main --template='{{ index .
 ----
 global:
   resolve_timeout: 5m
+  http_config:
+    proxy_from_environment: true # <1>
 route:
-  group_wait: 30s #<1>
-  group_interval: 5m #<2>
-  repeat_interval: 12h #<3>
+  group_wait: 30s # <2>
+  group_interval: 5m # <3>
+  repeat_interval: 12h # <4>
   receiver: default
   routes:
   - matchers:
     - "alertname=Watchdog"
     repeat_interval: 2m
     receiver: watchdog
   - matchers:
-    - "service=<your_service>" #<4>
+    - "service=<your_service>" # <5>
     routes:
     - matchers:
-      - <your_matching_rules> #<5>
-      receiver: <receiver> #<6>
+      - <your_matching_rules> # <6>
+      receiver: <receiver> # <7>
 receivers:
 - name: default
 - name: watchdog
 - name: <receiver>
-  <receiver_configuration> #<7>
+  <receiver_configuration> # <8>
 ----
-<1> Specify how long Alertmanager waits while collecting initial alerts for a group of alerts before sending a notification.
-<2> Specify how much time must elapse before Alertmanager sends a notification about new alerts added to a group of alerts for which an initial notification was already sent.
-<3> Specify the minimum amount of time that must pass before an alert notification is repeated.
+<1> If you configured an HTTP cluster-wide proxy, you can set the `proxy_from_environment` parameter to `true` to enable proxying for all alert receivers.
```

Comment:
```suggestion
<1> If you configured an HTTP cluster-wide proxy, set the `proxy_from_environment` parameter to `true` to enable proxying for all alert receivers.
```
from same description in https://github.com/openshift/openshift-docs/pull/85113/files#diff-9d3a4011173601a722946389c2d8bd36d09c69420017ae76e66ae45fe5e501beR50